getting-started/templates/systemlink-values.yaml

```diff
@@ -1082,6 +1082,39 @@ nbexecservice:
         # trustedCertificateSecret:
         #   secretName: S3Certificate
         #   key: cert
+    workflow:
+      run:
+        # Default resource allocation for execution pods
+        resources:
+          requests:
+            cpu: "0.1"
+            memory: 2Gi
+          limits:
+            memory: 2Gi
+        # Resource profiles enable different resource allocations for each execution. Profile names are fixed and cannot be modified.
+        # Use dedicated nodes to ensure that the scheduling of non-execution pods is not impacted.
```

Comment:
Can we reference https://github.com/ni/install-systemlink-enterprise/blob/main/getting-started/templates/node-selectors.yaml here? This shows how to keep notebook execution on a dedicate pod.